build/fbcode_builder/docker_build_with_ccache.sh

```diff
@@ -66,7 +66,7 @@ if [[ "$build_timeout" != "" ]] ; then
     echo "Build timed out after $build_timeout" 1>&2
     while true; do
       maybe_container=$(
-        egrep '^( ---> Running in [0-9a-f]+|FBCODE_BUILDER_EXIT)$' "$logfile" |
+        grep -E '^( ---> Running in [0-9a-f]+|FBCODE_BUILDER_EXIT)$' "$logfile" |
```

Comment:
Please split out the changes in this file to a separate PR.

Internally this file actually come from a separate project which gets auto-synced into the github versions of folly, thrift, and other projects.

Reply:
Thanks for the info. I made this PR just the change for `docker_build_with_ccache.sh` and then the other file is out for PR here: https://github.com/facebook/folly/pull/1058